examples/async/async.bal

```diff
@@ -12,12 +12,15 @@ public function main() {
     // You can pass the value of the `future` variable around
     // and call its results later.
     int result = squarePlusCube(f1);
+    // Wait for future `f1` to finish.
+    _ = wait f1;
     io:println("SQ + CB = ", result);
 
-    // Call the `countInfinity()` function, that runs forever in asynchronous 
+    // Call the `countInfinity()` function, that runs forever in asynchronous
     // mode.
     future<()> f2 = start countInfinity();
-    runtime:sleep(1000);
+    // A `runtime:sleep` is added to delay the execution.
+    runtime:sleep(2000);
```

Comment:
Shall we avid using sleeps for passing tests? Using a sleep is not a permanent solution. It should we used only if there is no other solution. Since this is a example test, what you can do is add all the possible results due to the parallel behavior, to the test.

Reply:
Or, you can join all the returned outputs and check each line exists.

Reply:
Fixed with [dd5ce2b](https://github.com/ballerina-platform/ballerina-lang/commit/dd5ce2b5a90f3a614a31d74ffe9360b8a90b7986). Thanks :)